admin/apple-actions/index/class-export.php

```diff
@@ -55,9 +55,6 @@ public function __construct( $settings, $id = null, $sections = null ) {
 		parent::__construct( $settings );
 		$this->set_theme( $sections );
 		$this->id = $id;
-
-		// Assign instance of an active Jetpack tiled gallery installation.
-		$jetpack_tiled_gallery = Jetpack_Tiled_Gallery::instance();
```

Comment:
This cannot be removed, otherwise the functionality in `includes/apple-exporter/third-party/class-jetpack-tiled-gallery.php` never gets initialized. If it's throwing errors in the latest phpcs, then we need an ignore rule here.

Reply:
I've reverted the commit that removes this line.